scripts/Ball.gd

```diff
@@ -14,9 +14,11 @@ func _physics_process(delta):
 	if is_instance_valid(collision):
 		direction = direction.bounce(collision.get_normal())
 
-		if collision.get_collider().is_in_group("block"):
-			collision.get_collider().on_collide_with_ball()
-	update()
```

Comment:
i thought this update was needed for _draw to work properly but it's not

Reply:
It would be if we were changing what was being drawn, I think, but because it's drawn and not altered again it doesn't matter

Reply:
aahhhhhh